Rename linalg.det test so that it generates a valid method name

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #53973 Restore storage on meta tensors; increase meta coverage
* #54711 Compute Tensor::toString() without reference to backend
* #54710 Make memory overlap a little less precise so it works with null data ptr
* #54709 Make the fuser raise NotImplementedError when unknown device is hit
* #54708 Avoid testing device in cdist when called in a "Math" context
* #54707 Make convolution_overrideable default implementation raise NotImplementedError
* #54706 Disambiguate where 'Doesn't run' error message comes from
* #54705 Fix missing format string qualifier
* **#54704 Rename linalg.det test so that it generates a valid method name**
* #54703 Add empty_generic, which allocates an empty tensor in a device-generic way
* #54597 Fix some more native_functions.yaml mistakes
* #54627 Nasty little hack to preserve NotImplementedError raised in interpreter

See https://github.com/pytorch/pytorch/issues/54607

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D27338809](https://our.internmc.facebook.com/intern/diff/D27338809)

ezyang committed:
Rename linalg.det test so that it generates a valid method name

See https://github.com/pytorch/pytorch/issues/54607

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

mruberry reviewed (approved):
Thank you! Note in the future this will be mooted because OpInfos now support aliasing:

https://github.com/pytorch/pytorch/blob/dfc7fa03e5d33f909b9d7853dd001086f5d782a0/torch/testing/_internal/common_methods_invocations.py#L2111

And the generated tests use a different naming scheme.

ezyang committed:
Update on "Rename linalg.det test so that it generates a valid method name"


See https://github.com/pytorch/pytorch/issues/54607

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D27338809](https://our.internmc.facebook.com/intern/diff/D27338809)

[ghstack-poisoned]
